pcsx2/gui/FrameForGS.cpp

```diff
@@ -171,6 +182,13 @@ void GSPanel::DoResize()
 	float zoom = g_Conf->GSWindow.Zoom.ToFloat()/100.0;
 	if( zoom == 0 )//auto zoom in untill black-bars are gone (while keeping the aspect ratio).
 		zoom = std::max( (float)arr, (float)(1.0/arr) );
+	if (arr == 1 && EmuConfig.Gamefixes.FMVinSoftwareHack && g_Conf->GSWindow.IsFullscreen) {
+		//avoids crash on some systems switching HW><SW in fullscreen aspect ratio's with FMV Software switch.
+		if (zoom < 1.001 && zoom >= 1.0000)
```

Comment:
Could you switch the order to `zoom >= 1.0000 && zoom < 1.001` ? It's slightly easier to read that way.

And could you squash the 3rd commit into your 2nd commit? I can merge after that's done.